plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -26,7 +26,7 @@
             <tr {% if plugin.isActivated|default(false) %}class="active-plugin"{% else %}class="inactive-plugin"{% endif %}>
                 <td class="select-cell">
                     <span class="checkbox-container">
-                        <input type="checkbox" id="select-plugin-{{ plugin.name|e('html_attr') }}" {% if plugin.isDownloadable is defined and not plugin.isDownloadable %}disabled="disabled"{% endif %} />
+                        <input type="checkbox" id="select-plugin-{{ plugin.name|e('html_attr') }}" onclick="boxes_selected()" {% if plugin.isDownloadable is defined and not plugin.isDownloadable %}disabled="disabled"{% endif %} />
```

Comment:
@MrYamous it doesn't work for me because of an error when I click on it that this method is not defined... but even when removing this onclick it still seems to not activate the "update selected" button.

Reply:
@tsteur According to diosmosis's feedback, i removed this onclick and boxes_selectes(). Did you test the last commit ?

Reply:
There is still `onclick="boxes_selected()"` though but no such method. Maybe you forgot to commit something?

Reply:
Exact, i missed one onclick, fixed in a new commit